authorizer-app/src/app/admin/services/user.service.ts

```diff
@@ -68,4 +69,28 @@ export class UserService {
     );
     return this.http.delete(url);
   }
+
+  storeUserAuthParams(url: string) {
```

Comment:
I see this used to be present in 3.x.x versions -- https://github.com/RADAR-base/RADAR-Rest-Source-Auth/blob/97fd30092b4e52a1f5ad7e26783f1d76cd6941db/authorizer-app/src/app/services/source-client-authorization.service.ts#L48

I wonder why it was removed during refactoring.